src/Loader.cpp

```diff
@@ -41,6 +41,31 @@ int main_loader(void)
             // TODO: Combo (X + LEFT) - Force 420P output
             // TODO: Combo (X + UP) - Force 720P output
             // TODO: Combo (X + RIGHT) - Force 1080i output
+
```

Comment:
You should probably add a `}` here.

Reply:
That should do it, I knew I had to be missing one somewhere, but I just couldn't see it. Thanks.